packages/core/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@blockprotocol/core",
```

Comment:
If a folder for `@blockprotocol/x` is `packages/x`, it ends up being a sibling of `packages/create-block-app` and alike. This becomes somewhat confusing. Folder suggestions for `@blockprotocol/x`:

- `packages/blockprotocol--x` (with `--` like for `@types/namespace--package` on NPM)
- `packages/@blockprotocol/x` (requires a adding `packages/@blockprotocol/*` to root `package.json` → `workspaces`)

Either way, it’d be nice to have _some_ way of unambiguously mapping folder names to package names or vice versa.

Reply:
Moved in https://github.com/blockprotocol/blockprotocol/pull/323/commits/1660342ca9dd945376f882e19b2f139d7540b958